libs/dal/src/repositories/subscriber/subscriber.repository.ts

```diff
@@ -46,14 +46,29 @@ export class SubscriberRepository extends BaseRepository<SubscriberDBModel, Subs
     });
   }
 
-  async searchSubscribers(environmentId: string, search?: string | null, emails: string[] = []) {
-    const filters: any = [
-      {
+  async searchSubscribers(
+    environmentId: string,
+    subscriberIds?: string[] | null,
```

Comment:
```suggestion
    subscriberIds?: string[] = [],
```
The same as we do with emails.

Reply:
Good catch